code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -631,7 +631,7 @@
 		return
 	var/list/ai_emotions = list("Very Happy", "Happy", "Neutral", "Unsure", "Confused", "Sad", "BSOD", "Blank", "Problems?", "Awesome", "Facepalm", "Friend Computer", "Dorfy", "Blue Glow", "Red Glow")
 	var/emote = input("Please, select a status!", "AI Status", null, null) in ai_emotions
-	for (var/obj/machinery/ai_status_display/M in GLOB.ai_status_displays) //change status of displays
+	for (var/obj/machinery/status_display/ai/M in GLOB.ai_status_displays) //change status of displays

```

Comment:
There are non ai status displays in GLOB.ai_status_displays?

Reply:
Before this PR all status display were included in `GLOB.ai_status_displays`, as of now it's probably safe to make this a non-typechecked loop